src/app/(bluerpint-list)/BlueprintList.tsx

```diff
@@ -0,0 +1,126 @@
+'use client';
+
+import Link from 'next/link';
+import BlueprintCard from '@/app/components/BlueprintCard';
+import zkeSdk, { Blueprint } from '@dimidumo/zk-email-sdk-ts';
+import { useState, useEffect, useRef, useCallback } from 'react';
+
+interface BlueprintListProps {
+  search: string | null;
+}
+
+export default function BlueprintList({ search }: BlueprintListProps) {
+  const [blueprints, setBlueprints] = useState<Blueprint[]>([]);
+  const [skip, setSkip] = useState(0);
+  const [limit] = useState(10);
```

Comment:
Convert it to constant as it never changes. We can change it later if we add support option to change pagination limit

Reply:
Blind ChatGPT